types/plotly.js/index.d.ts

```diff
@@ -842,8 +842,16 @@ export type Calendar =
     | "thai"
     | "ummalqura";
 
-export type XAxisName = "x" | "x2" | "x3" | "x4" | "x5" | "x6" | "x7" | "x8" | "x9" | "x10" | "x11";
-export type YAxisName = "y" | "y2" | "y3" | "y4" | "y5" | "y6" | "y7" | "y8" | "y9" | "y10" | "y11";
+// regex from documentation: "/^x([2-9]|[1-9][0-9]+)?( domain)?$/" | "/^y([2-9]|[1-9][0-9]+)?( domain)?$/"
+// regex allows for an unlimited amount of digits for the 'axis number', but the following typescript definition is limited to two digits
+type xYAxisNames = `${
+    | ""
+    | `${2 | 3 | 4 | 5 | 6 | 7 | 8 | 9}`
+    | `${1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9}${0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9}`}${"" | " domain"}`;
+
+export type XAxisName = `${"x"}${xYAxisNames}`;
+export type YAxisName = `${"y"}${xYAxisNames}`;
```

Comment:
```suggestion
export type XAxisName = `x${xYAxisNames}`;
export type YAxisName = `y${xYAxisNames}`;
```

Not sure this needs to have these extra ones.